docs/kick/kick.md

```diff
@@ -0,0 +1,295 @@
+# Browser beats I: synthesizing a kick drum
+
+Because I wanted to gain some experience in working with [Angular](https://angular.io/) and [NgRx](https://ngrx.io/), I started building a sample-based step sequencer that runs in the browser.
+To do that, I had to dive into the [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API).
+I'll write something about that step sequencer later.
+First, I want to take a look at the basics of the Web Audio API and try to synthesize a kick drum.
+
+## The basis
+
+At the basis of most syntesized kick drums, there's a sine wave, or something that's close to a sine wave.
+The function below produces a sine wave with a frequency of 55Hz that plays for the duration of ten seconds.
+
+```js
+const play = () => {
+    const audioContextClass = window.AudioContext || window.webkitAudioContext;
+    const audioContext = new audioContextClass();
+
+    const sine = audioContext.createOscillator();
+    sine.type = 'sine';
+    sine.frequency.value = 55;
+
+    sine.start(audioContext.currentTime);
+    sine.stop(audioContext.currentTime + 10);
+}
+```
+
+It sounds like this:
+(You might not hear it without head phones.)
+
+<audio controls src="audio/sine-55Hz.mp3"></audio>
+
+When you visualize that sound, as shown below, you'll see why it's called a sine wave.
+The left-hand side of the figure shows the waveform, and the right-hand side shows the [sound spectrum](https://newt.phys.unsw.edu.au/jw/sound.spectrum.html).
+
+![Sine at 55Hz](graphs/sine-55Hz.png)
+
+The sound spectrum is almost completely empty, except for a narrow spike at the rightmost end.
+This explains why you might not hear the sound over your laptop speakers, for example.
+Not all speakers are capable of reproducing sounds at low frequencies.
+You can emulate the frequency response of such speakers by applying a high-pass filter.
```

Comment:
A link that explains what a high pass filter is might be helpful for some readers.